service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/EventsCheckerTest.java

```diff
@@ -141,21 +130,6 @@ void shouldCheckEventIsAvailable(Event event, EventChecker validator) {
         verify(validator).isAvailable(caseData);
     }
 
-    @ParameterizedTest
-    @MethodSource("getEventsValidatorsWithApplicationDocumentsToggledOn")
-    void shouldNotValidateDocumentEventWhenApplicationEventToggledOn(Event event, EventChecker validator) {
-        given(featureToggleService.isApplicationDocumentsEventEnabled()).willReturn(true);
-
```

Comment:
Would we not want this test to stay to verify no validation was being called on this event?

Reply:
We wouldnt want the entire test to stay but we could append the -
`verify(documentsChecker, never()).validate(any());`

to an existing?

Reply:
yeah maybe it's fine actually, if we have another test that verifies the correct checkers are collect.